### DISCUSSION TITLE
Field file multiple

### DISCUSSION THREAD

**Discussion by Polm90** [joomengine/Joomla-Component-Builder#783] [Q&A] [answered]
Hi,
I was wondering why when creating a field with type "file" there is not the option to set the "multiple" property (currently supported by Joomla as you can see in the documentation https://docs.joomla.org/File_form_field_type) 

Thank you!

**Comment by Llewellynvdm** [comment, marked as answer]
You can add more properties if you want, just open the field type, and add more properties you need.
